clustered/client/src/main/java/org/ehcache/clustered/client/internal/config/xml/ClusteredResourceConfigurationParser.java

```diff
@@ -104,11 +125,42 @@ public ResourcePool parseResourceConfiguration(final Element fragment) {
 
   @Override
   public Element unparseResourcePool(ResourcePool resourcePool) {
-    throw new UnsupportedOperationException("Not yet implemented");
+    return unparseConfig(resourcePool);
+  }
+
+  @Override
+  public Element createRootElement(Document doc, ResourcePool resourcePool) {
+    Element rootElement = doc.createElementNS(EHCACHE_NAMESPACE, EHCACHE_NAMESPACE_PREFIX + COLON + RESOURCE_ELEMENT_NAME);
```

Comment:
Why is a service parser creating an element in the ehcache core namespace?  The calling code should be creating this element for us.